### ISSUE TITLE
Template string arguments are merging into one

### ISSUE THREAD

**Issue by kot-shrodingera** [javascript-obfuscator/javascript-obfuscator#716]
Code example
```
const tag = (strings, ...args) => {
  return strings.raw
};

tag`begin${'first'}middle${'second'}end`;
```
Expected output is `["begin", "middle", "end"]` (with args[0] = "first" and args[1] = "middle")
But current output is `["", ""]` (with args[0] = "beginfirstmiddlesecondend")
This is because code is obfuscating into this (with simplest settings):
```
const tag = (a, ...b) => {
    return a['raw'];
};
tag`${ 'begin' + 'first' + 'middle' + 'second' + 'end' }`;
```

I'm using tag functions and need to distinguish parts of template strings, but with such behavior it doesn't work
Sorry for bad English

**Comment by sanex3339** [comment]
Ohh. Looks like i have to revert https://github.com/javascript-obfuscator/javascript-obfuscator/issues/696

**Comment by sanex3339** [comment]
I'll release a new release with disabled `tagged template literals` obfuscation in a few hours.

**Comment by sanex3339** [comment]
Released `1.11.0` with disabled `tagged template literals` obfuscation.
